src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+server.port=8081
```

Comment:
@MarkoLazovicCode As I mentioned, we should have a couple of environment(s) definitions. One for production, and one for example for development.